Sources/StreamChatUI/CommonViews/Attachments/AttachmentViews/FileAttachmentView.swift

```diff
@@ -89,7 +90,9 @@ open class FileAttachmentView: _View, AppearanceProvider {
             ),
             fileNameAndSizeStack.centerYAnchor.pin(equalTo: centerYAnchor),
             fileNameAndSizeStack.topAnchor.pin(greaterThanOrEqualTo: layoutMarginsGuide.topAnchor),
-            fileNameAndSizeStack.bottomAnchor.pin(lessThanOrEqualTo: layoutMarginsGuide.bottomAnchor)
+            fileNameAndSizeStack.bottomAnchor.pin(lessThanOrEqualTo: layoutMarginsGuide.bottomAnchor),
+            // This is to avoid the file name to go under the "X" Close button, which doesn't belong to this view.
+            fileNameAndSizeStack.trailingAnchor.pin(equalTo: layoutMarginsGuide.trailingAnchor, constant: -30)
```

Comment:
Does this cover bigger device sizes? Landscape/Portrait?

Reply:
Yeah, that part with a magic constant is a bit weird, tbh. Would it work if we pin `preview.trailingAnchor` to discard button leading anchor when embedding the preview into the container? 🤔

Reply:
@adamrushy @evsaev Yes it works on landscape as well, since the hard coded value is still related to the trailingAnchor, is not a constant width. The preview container doesn't have any information about what it contains, so we can't add the constraint in a layer above. For now, this is the best solution and it works 👍